### DISCUSSION TITLE
Are there any api you can use to create a provider for a data base that prisma dosen't support yet???

### DISCUSSION THREAD

**Discussion by David00154** [prisma/prisma#12805] [Q&A]
I want to create a custom provider that prisma dosen't support yet but I cant see any where in the docs on how to

**Comment by austincrim** [comment]
Hey @David00154 👋 ,

We do not expose a user-facing API to build custom database providers. What provider are you interested in supporting? We might have an existing feature request for it or we can [create a new one](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
